lib/node_modules/@stdlib/blas/ext/base/ndarray/dapx/benchmark/benchmark.js

```diff
@@ -0,0 +1,108 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
+var isnanf = require( '@stdlib/math/base/assert/is-nanf' );
+var pow = require( '@stdlib/math/base/special/pow' );
+var ndarray = require( '@stdlib/ndarray/ctor' );
+var scalar2ndarray = require( '@stdlib/ndarray/from-scalar' );
+var pkg = require( './../package.json' ).name;
+var dapx = require( './../lib' );
+
+
+// FUNCTIONS //
```

Comment:
```suggestion
// VARIABLES //

var options = {
	'dtype': 'float64'
};


// FUNCTIONS //
```

Prefer parameterization to limit copy-paste mistakes in other packages. This is one reason to first do one package (e.g., `dapx`) before moving on to other packages (e.g., `sapx`, `gapx`), as it reduces the number of changes you need to make. Currently, you are having to replicate changes.